src/swap/SwapScreen.tsx

```diff
@@ -225,14 +225,11 @@ export function SwapScreen({ route }: Props) {
 
   const handleShowTokenSelect = (fieldType: Field) => () => {
     ValoraAnalytics.track(SwapEvents.swap_screen_select_token, { fieldType })
-    // ensure that the keyboard is dismissed before animating token bottom sheet
-    Keyboard.dismiss()
     setSelectingToken(fieldType)
-    tokenBottomSheetRef.current?.snapToIndex(0)
-  }
 
-  const handleCloseTokenSelect = () => {
```

Comment:
this was redundant, since the selecting token is already cleared when the token is selected in `handleSelectToken`